lib/model/yake_keywords.py

```diff
@@ -26,15 +43,31 @@ def run_yake(self, text: str,
         :param num_of_keywords: int
         :returns: str
         """
+        ### if language is set to "auto", auto-detect it.
```

Comment:
I still think we need unit tests for the replacement/substitution part (because this list is likely to grow a lot).  so maybe worth a tiny refactor to put this part in its own function so easy to test